src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -52,6 +52,15 @@ public CantGetEditAssetsException(String message, Throwable cause) {
         }
     }
 
+    public static class Edit {
+        private Edit(ViewVisitor aVisitor, List<String> someUrls) {
```

Comment:
Some Java codebases prefix their arguments with "a"- I'm doing it here (And do it elsewhere) when the natural argument name would be the same as a public member name, as the compliment to the convention of prefixing "m" to member names (since we don't do that with public members). I used "some" rather than "a" for URLS for grammar (because aUrls seems strange)

The alternative would be something like
```
private Edit(ViewVisitor visitor, List<String> urls) {
    this.visitor = visitor;
    this.urls = urls;
}
```
Which I don't like as much, since the this looks redundant (and usually is redundant)